src/main/java/domain/money/Money.java

```diff
@@ -0,0 +1,27 @@
+package domain.money;
+
+import domain.game.Result;
+
+public class Money {
+    private final int value;
+
+    public Money(String value) {
+        try {
+            this.value = Integer.parseInt(value);
+        } catch (NumberFormatException ne) {
+            throw new NumberFormatException("양의 정수만 입력하세요.");
+        }
+        validate();
```

Comment:
```suggestion
        try {
            int value = Integer.parseInt(rawValue);
            validate(value);
            this.value = value;
        } catch (NumberFormatException ne) {
            throw new NumberFormatException("양의 정수만 입력하세요.");
        }
```
validate가 try-catch 밖에 있어서 놓치기 쉬울것 같네요

Reply:
감사합니다. 모든 검증 이후에 값을 넣어 주는게 더 자연스러운 것 같습니다!